### PR TITLE
Make server more helpful

### PR DESCRIPTION
**Pull request by keawade**

### CODE REVIEW

File: routes/auth.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ module.exports = function (express) {
   let router = express.Router()
   router.route('/signup')
     .post(function (req, res, next) {
-      console.log(`[garden] POST /api/signup`, req.body)
+      console.log(`[garden] POST /api/signup`, req.body.username)
```
Comment (NathanBland) [inline routes/auth.js:9]: :+1:

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,603 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (NathanBland) [inline yarn.lock:1]: I see you guys trying to sneak this in while I'm away.... -.-

Comment (keawade) [reply yarn.lock:1]: Not trying and not sneaking. 😛

Comment (crodeheaver) [reply yarn.lock:1]: @keawade Told you he wouldn't like it. haha. ;P

**Pull request merged**